rotkehlchen/exchanges/exchange.py

```diff
@@ -507,3 +509,19 @@ def query_history_with_callbacks(
             )
         except RemoteError as e:
             fail_callback(str(e))
+
+    def send_unknown_asset_message(
+            self,
+            msg_aggregator: 'MessagesAggregator',
+            asset_identifier: str,
+            details: str,
+    ) -> None:
+        log.warning(f'Found unknown {self.name} asset {asset_identifier} in {details}.')
+        msg_aggregator.add_message(
+            message_type=WSMessageType.EXCHANGE_UNKNOWN_ASSET,
+            data={
+                'location': self.location.serialize(),
```

Comment:
Also add the name here and in the websocket and its api docs. What is the name? The name is the name the user can give to the specific api key instance.

What does that mean? It's useless in most cases as the name is default (such as kraken-1) but it can be used to differentiate between two different api keys for two different accounts on the same exchange. Say you have two coinbase accounts. 

```suggestion
                'location': self.location.serialize(),
                'name': self.name
```